crates/rolldown_common/src/types/stmt_info.rs

```diff
@@ -97,22 +98,30 @@ pub struct StmtInfo {
   pub side_effect: bool,
   pub is_included: bool,
   pub import_records: Vec<ImportRecordIdx>,
+  #[cfg(debug_assertions)]
   pub debug_label: Option<String>,
   pub meta: StmtInfoMeta,
 }
 
+#[cfg(target_pointer_width = "64")]
+const _: () = {
+  #[cfg(not(debug_assertions))]
+  assert!(size_of::<StmtInfo>() == 88usize);
```

Comment:
from 120 to 88 in release mode, this could save a mount of memory in release mode, since stmt info count = (module count * stmt info cout)